config/jobs/kubernetes/sig-testing/ec2-e2e.yaml

```diff
@@ -120,6 +180,65 @@ presubmits:
             requests:
               cpu: 8
               memory: 10Gi
+  - name: pull-kubernetes-e2e-ec2-conformance-arm64
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+      testgrid-dashboards: presubmits-ec2
```

Comment:
Can we have a description here for this job like "pull-kubernetes-e2e-ec2-arm64"?

Reply:
Done